apple/RNCWebView.m

```diff
@@ -185,14 +187,119 @@ - (instancetype)initWithFrame:(CGRect)frame
 
   }
 #endif // !TARGET_OS_OSX
+
+  // Allow this object to recognize gestures
+  UILongPressGestureRecognizer *longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(startLongPress:)];
+  longPress.delegate = self;
+
+  longPress.minimumPressDuration = 0.4f;
+  longPress.numberOfTouchesRequired = 1;
+  longPress.cancelsTouchesInView = YES;
+  [self addGestureRecognizer:longPress];
```

Comment:
The checks you added are good, but I was _kind of hoping_ that even the gesture recognizer could be skipped if the menu item properties weren't set.

I'm not sure how complicated that would be though, or how likely it is that this could actually cause any trouble, it just makes me slightly nervous that this new gesture recognizer would be in place for everybody even if they're not using this new feature...

Reply:
I'm really not overly familiar with Objective-C or the lifecycle of these components.  `self.menuItems` isn't defined at this point, so I can't do a check on whether or not the object exists.  Is there another lifecycle hook I can use that would have the same effect?